hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/table/TestCleaner.java

```diff
@@ -111,6 +112,8 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
+import scala.Tuple3;
```

Comment:
anyway we can avoid this?

Reply:
This is due to autoformatting from Save action.